package.json

```diff
@@ -17,23 +17,25 @@
     "@codesandbox/sandpack-themes": "^1.17.0",
     "@next/bundle-analyzer": "^13.1.1",
     "@radix-ui/react-tabs": "^1.0.1",
-    "@tanstack/react-query": "^4.19.1",
     "framer-motion": "^7.8.0",
-    "luxon": "^3.1.1",
     "next": "13.0.6",
     "react": "18.2.0",
     "react-dom": "18.2.0",
     "react-focus-on": "^3.7.0",
     "react-lottie-player": "^1.5.4",
     "react-mailchimp-subscribe": "^2.1.3",
-    "styled-components": "^5.3.6"
+    "sharp": "^0.31.3",
```

Comment:
the nextjs `<Image>` component automatically optimizes images at build time with `sharp`